lib/commands/context.js

```diff
@@ -25,18 +26,10 @@ commands.getCurrentContext = async function getCurrentContext () { // eslint-dis
 };
 
 commands.getContexts = async function getContexts () {
-  let webviews;
-  if (this.isChromeSession) {
-    // if we have a Chrome browser session, we only care about the Chrome
-    // context and the native context
-    webviews = [CHROMIUM_WIN];
-  } else {
-    // otherwise we use ADB to figure out which webviews are available
-    webviews = await webviewHelpers.getWebviews(this.adb, this.opts);
-  }
-  this.contexts = _.union([NATIVE_WIN], webviews);
-  log.debug(`Available contexts: ${JSON.stringify(this.contexts)}`);
-  return this.contexts;
+  const webviewsMapping = this.isChromeSession
+    ? []
+    : await webviewHelpers.getWebViewsMapping(this.adb, this.opts);
```

Comment:
[nits]
Can extract duplicated lines here and L47-L49

Reply:
Addressed as https://github.com/appium/appium-android-driver/pull/699/commits/6e5c4929ce89493a6fac348b0cad1051363b04bb